src/java/org/apache/cassandra/utils/MBeanWrapper.java

```diff
@@ -170,6 +186,125 @@ public void unregisterMBean(ObjectName mbeanName, OnException onException)
         }
     }
 
+    static class InstanceMBeanWrapper implements MBeanWrapper
+    {
+        private final MBeanServer mbs;
+        public final UUID id = UUID.randomUUID();
+
+        public InstanceMBeanWrapper(String hostname)
+        {
+            mbs = MBeanServerFactory.createMBeanServer(hostname + "-" + id);
+        }
+
+        public void registerMBean(Object obj, ObjectName mbeanName, OnException onException)
```

Comment:
can we add the `@Override` annotation to distinguish between internal methods and overridden methods?

Reply:
Yeah - none of the other existing ones had `@Override` annotations before so I didn't add them here... I can just add them to everything so it's at least consistent.

Reply:
Decided to leave off the `@Override`s for now as it would spam the review with noise. If desired, we could do a follow-up to enforce the code style on files not currently following it.